Free minters (#251)
soundxyz/sound-protocol: README.md

## Deployments

> All contracts are deployed on Goerli testnet and Mainnet.

Reviewer: ```suggestion
Permissionless zero-fee deployments on Goerli testnet and Mainnet:
```

| Contract  |  Address |
|---|---|

> | `SoundFeeRegistry` | 0x8f921211c9771baeb648ac7becb322a540298a4b |

Reviewer: ```suggestion
```
